pkgs/development/python-modules/torch/default.nix

```
@@ -52,15 +52,15 @@
 
 let
   inherit (lib) lists strings trivial;
-  inherit (cudaPackages) cudatoolkit cudaFlags cudnn nccl;
+  inherit (cudaPackages) cudaFlags cudnn nccl;
 in
 
 assert cudaSupport -> stdenv.isLinux;
 assert cudaSupport -> (cudaPackages.cudaMajorVersion == "11");
 
 # confirm that cudatoolkits are sync'd across dependencies
-assert !(MPISupport && cudaSupport) || mpi.cudatoolkit == cudatoolkit;
-assert !cudaSupport || magma.cudaPackages.cudatoolkit == cudatoolkit;
+assert !(MPISupport && cudaSupport) || mpi.cudatoolkit == cudaPackages.cudatoolkit;
+assert !cudaSupport || magma.cudaPackages.cudaVersion == cudaPackages.cudaVersion;
```

```suggestion
assert (MPISupport && cudaSupport) -> mpi.cudatoolkit == cudaPackages.cudatoolkit;
assert cudaSupport -> magma.cudaPackages.cudaVersion == cudaPackages.cudaVersion;
```
not blocking, but while we're here

---

Fixed in latest